examples/lib/pods.rego

```diff
@@ -2,32 +2,21 @@ package lib.pods
 
 import data.lib.core
 
-pods[pod] {
-    lower(core.kind) == "statefulset"
-    pod = core.resource.spec.template
+pod = core.resource.spec.template {
+    pod_templates := ["daemonset","deployment","job","statefulset"]
```

Comment:
We probably should have done this earlier, but we should add `replicaset` as well. Even though `Depoyment`s are standard now, we don't want folks to bypass our policies -- especially our security policies -- by using a different (edit: but still valid) resource type. 

We're also missing support for CronJob resources but those use a different format than `resource.spec.template`.

Edit again: ReplicationController, added in 1.18, should also be included.

Reply:
Done!